src/components/Article.tsx

```diff
@@ -1,87 +1,629 @@
-import Image from "next/image";
-import Link from "next/link";
-import SVG from "react-inlinesvg";
-import { AiOutlineUser } from "react-icons/ai";
-import { StrapiImageFormats } from "@models/types";
-import format from "date-fns/format";
+import Image from 'next/image'
+import Link from 'next/link'
+import SVG from 'react-inlinesvg'
+import { AiOutlineUser } from 'react-icons/ai'
+import { StrapiImageFormats } from '@models/types'
+import format from 'date-fns/format'
 
 interface ArticleProps {
-  title: string;
-  excerpt: string;
-  author: string;
-  date: string;
-  slug: string;
-  image: StrapiImageFormats;
+  title: string
+  excerpt: string
+  author: string
+  date: string
+  slug: string
+  image: StrapiImageFormats
 }
 
 export default function Article({
-  title = "Guide name",
-  excerpt = "Lorem ipsum dolor sit amet",
-  author = "author",
-  date = "2021-08-28T00:51:14.244Z",
-  slug = "getting-started",
+  title = 'Guide name',
+  excerpt = 'Lorem ipsum dolor sit amet',
+  author = 'author',
+  date = '2021-08-28T00:51:14.244Z',
+  slug = 'getting-started',
   image,
 }: ArticleProps): JSX.Element {
   return (
     <Link href={`/guide/${slug}`} passHref={true}>
-      <a className="group">
-        <article className="article-parent relative">
-          <div className="diamond-border">
-            <div className="article__mask absolute top-0 left-32 select-none ">
-              <SVG src="/assets/images/icons/mask.svg" />
-              <div className="losange h-16 w-16 sm:h-32 sm:w-32 lg:h-64 lg:w-64">
-                <Image
-                  objectFit="cover"
-                  src={
-                    image?.large?.url ??
-                    image?.medium?.url ??
-                    image?.small?.url ??
-                    image?.thumbnail?.url
-                  }
-                  height={512}
-                  width={512}
-                  alt={`${title} thumbnail`}
+      <a className='group'>
+        <article className=''>
+          <svg
```

Comment:
Can this be refactored into a separate component in the `decorations` folder ? so we can use it like this :

```jsx
<LineWithSquares image={image} />
```